RAMP/unpacker.py

```diff
@@ -27,14 +26,18 @@ def __str__(self):
 
 
 def unpack(bundle, max_bundle_size_mb=None):
-    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes]]
+    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes], Dict[str, Any]]
     """
     Unpacks a bundled module, performs sanity validation on bundle.
     both the module metadata and the actual module are returned
     :rtype: tuple
     """
+    deps_files = dict()
     try:
         with ZipFile(bundle) as zf:
+            for filename in zf.namelist():
```

Comment:
I feel like this should come after "validate_zip_file"
right after `module=; metadata=;`